modules/mobile/spec/request/message_drafts_request_spec.rb

```diff
@@ -2,13 +2,11 @@
 
```

Comment:
These spec changes are hard to read, but I generally see what you're doing and it makes sense. The auth aspect of the messaging controller would have been a really good candidate for an rspec shared example when the messaging request specs were originally implemented. I don't think it's worth refactoring for it at this point. Good work getting all of these changed. 👍